### ISSUE TITLE
The project doesn't build on iOS with the last Flutter's version

### ISSUE THREAD

**Issue by duduBaiao** [bizz84/starter_architecture_flutter_firebase#7]
This my Flutter version:

Flutter (Channel stable, v1.12.13+hotfix.8, on Mac OS X 10.15.3 19D76, locale en-DE)

This is the console output:

> flutter run
>  
> Launching lib/main.dart on iPhone de Eduardo in debug mode...
>  
> Automatically signing iOS for device deployment using specified development team in Xcode project: 9QYVXCXWY8
>  
> Running pod install...                                              9.9s
> Running Xcode build...                                                  
>                                                    
> Xcode build done.                                           15.4s
> Failed to build iOS app
> Error output from Xcode build:
> ↳
>     2020-02-18 23:25:42.746 xcodebuild[65756:818184]  DTDeviceKit: Failed to mount DDI from
>     '<DVTFilePath:0x7ffb2bc10dc0:'/Applications/Xcode.app/Contents/Developer/Platforms/WatchOS.platform/DeviceSupport/6.1/Devel
>     operDiskImage.dmg'>', error=-402653158: Error Domain=com.apple.dtdevicekit Code=-402653158 "The device is passcode
>     protected." UserInfo={NSLocalizedFailureReason=Please unlock your device and reattach. (0xE800001A).,
>     com.apple.dtdevicekit.stacktrace=(
>     	0   DTDeviceKitBase                     0x0000000110a0a6e7 DTDKCreateNSError + 109
>     	1   DTDeviceKitBase                     0x0000000110a0ade9 DTDK_AMDErrorToNSError + 792
>     	2   DTDeviceKitBase                     0x0000000110a48780 -[DTDKMobileDeviceToken _mountDeveloperDiskImage:withError:] +
>     	844
>     	3   DTDeviceKitBase                     0x0000000110a4929d __58-[DTDKMobileDeviceToken
>     	mountDeveloperDiskImageWithError:]_block_invoke + 699
>     	4   libdispatch.dylib                   0x00007fff6c15c50e _dispatch_client_callout + 8
>     	5   libdispatch.dylib                   0x00007fff6c168567 _dispatch_lane_barrier_sync_invoke_and_complete + 60
>     	6   DTDeviceKitBase                     0x0000000110a48f80 -[DTDKMobileDeviceToken mountDeveloperDiskImageWithError:] +
>     	199
>     	7   IDEiOSSupportCore                   0x000000011062e107 __28-[DVTiOSDevice attachToken:]_block_invoke_4 + 254
>     	8   DVTFoundation                       0x00000001013e9e7a __DVT_CALLING_CLIENT_BLOCK__ + 7
>     	9   DVTFoundation                       0x00000001013eb552 __DVTDispatchAsync_block_invoke + 809
>     	10  libdispatch.dylib                   0x00007fff6c15b583 _dispatch_call_block_and_release + 12
>     	11  libdispatch.dylib                   0x00007fff6c15c50e _dispatch_client_callout + 8
>     	12  libdispatch.dylib                   0x00007fff6c15e6c0 _dispatch_continuation_pop + 414
>     	13  libdispatch.dylib                   0x00007fff6c15ddbe _dispatch_async_redirect_invoke + 703
>     	14  libdispatch.dylib                   0x00007fff6c16a7e2 _dispatch_root_queue_drain + 326
>     	15  libdispatch.dylib                   0x00007fff6c16af22 _dispatch_worker_thread2 + 92
>     	16  libsystem_pthread.dylib             0x00007fff6c3b66b6 _pthread_wqthread + 220
>     	17  libsystem_pthread.dylib             0x00007fff6c3b5827 start_wqthread + 15
>     ), DVTRadarComponentKey=487927, NSLocalizedDescription=The device is passcode protected.}
>     2020-02-18 23:25:42.747 xcodebuild[65756:818184]  iPhoneSupport: 📱<DVTiOSDevice (0x7ffb2b1dbd80), iPhone de Eduardo,
>     iPhone, 13.3.1 (17D50), 9df66569dff84bc18b6fafb3710fae8698b14a30> unable to mount DeveloperDiskImage (Error
>     Domain=com.apple.dtdevicekit Code=-402653158 "The device is passcode protected." UserInfo={NSLocalizedFailureReason=Please
>     unlock your device and reattach. (0xE800001A)., com.apple.dtdevicekit.stacktrace=(
>     	0   DTDeviceKitBase                     0x0000000110a0a6e7 DTDKCreateNSError + 109
>     	1   DTDeviceKitBase                     0x0000000110a0ade9 DTDK_AMDErrorToNSError + 792
>     	2   DTDeviceKitBase                     0x0000000110a48780 -[DTDKMobileDeviceToken _mountDeveloperDiskImage:withError:] +
>     	844
>     	3   DTDeviceKitBase                     0x0000000110a4929d __58-[DTDKMobileDeviceToken
>     	mountDeveloperDiskImageWithError:]_block_invoke + 699
>     	4   libdispatch.dylib                   0x00007fff6c15c50e _dispatch_client_callout + 8
>     	5   libdispatch.dylib                   0x00007fff6c168567 _dispatch_lane_barrier_sync_invoke_and_complete + 60
>     	6   DTDeviceKitBase                     0x0000000110a48f80 -[DTDKMobileDeviceToken mountDeveloperDiskImageWithError:] +
>     	199
>     	7   IDEiOSSupportCore                   0x000000011062e107 __28-[DVTiOSDevice attachToken:]_block_invoke_4 + 254
>     	8   DVTFoundation                       0x00000001013e9e7a __DVT_CALLING_CLIENT_BLOCK__ + 7
>     	9   DVTFoundation                       0x00000001013eb552 __DVTDispatchAsync_block_invoke + 809
>     	10  libdispatch.dylib                   0x00007fff6c15b583 _dispatch_call_block_and_release + 12
>     	11  libdispatch.dylib                   0x00007fff6c15c50e _dispatch_client_callout + 8
>     	12  libdispatch.dylib                   0x00007fff6c15e6c0 _dispatch_continuation_pop + 414
>     	13  libdispatch.dylib                   0x00007fff6c15ddbe _dispatch_async_redirect_invoke + 703
>     	14  libdispatch.dylib                   0x00007fff6c16a7e2 _dispatch_root_queue_drain + 326
>     	15  libdispatch.dylib                   0x00007fff6c16af22 _dispatch_worker_thread2 + 92
>     	16  libsystem_pthread.dylib             0x00007fff6c3b66b6 _pthread_wqthread + 220
>     	17  libsystem_pthread.dylib             0x00007fff6c3b5827 start_wqthread + 15
>     ), DVTRadarComponentKey=487927, NSLocalizedDescription=The device is passcode protected.})
>     ** BUILD FAILED **
> 
> 
> Xcode's output:
> ↳
>     /Users/dudu/Documents/flutter-projects/starter_architecture_flutter_firebase/ios/Pods/FirebaseAuth/Firebase/Auth/Source/Aut
>     h/FIRAuth.m:32:9: fatal error: 'GoogleUtilities/GULSceneDelegateSwizzler.h' file not found
>     #import <GoogleUtilities/GULSceneDelegateSwizzler.h>
>             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
>     /Users/dudu/Documents/flutter-projects/starter_architecture_flutter_firebase/ios/Pods/FirebaseAuth/Firebase/Auth/Source/Aut
>     h/FIRAuth.m:32:9: note: did not find header 'GULSceneDelegateSwizzler.h' in framework 'GoogleUtilities' (loaded from
>     '/Users/dudu/Documents/flutter-projects/starter_architecture_flutter_firebase/build/ios/Debug-iphoneos/GoogleUtilities')
>     1 error generated.
>     note: Using new build system
>     note: Planning build
>     note: Constructing build description
> 
> Could not build the precompiled application for the device.

**Comment by bizz84** [comment]
Your log seems to show this:

> Error Domain=com.apple.dtdevicekit Code=-402653158 "The device is passcode
protected." UserInfo={NSLocalizedFailureReason=Please unlock your device and reattach. (0xE800001A).

Are you sure that this is not your problem?

**Comment by bizz84** [comment]
Also, there seems to be a good answer here:
https://stackoverflow.com/questions/59754797/googleutilities-gulscenedelegateswizzler-h-file-not-found
TL;DR: run `pod update`.
Let me know if this works.

**Comment by chi-tea-72** [comment]
Hi @duduBaiao ,

I agree with @bizz84's screen lock answer, but also just wanted to quickly mention the following, since I noticed you're running ios 13.3.1. There is an open issue on the flutter repo about building on ios 13.3.1+. It's really an apple issue, not a flutter issue: https://github.com/flutter/flutter/issues/49504.

**Comment by bizz84** [comment]
flutter/flutter#49504 is now resolved. Upgrading to iOS 13.4 should solve this.

@duduBaiao I'm closing this issue, but feel free to open a new one if this is still a problem.
